modules/kg-schemas/src/main/resources/schemas/nexus/core/datadownload/v0.1.0.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
I would recommend using `distribution` as the schema name instead and update all shape ids to be consistent.